app/controllers/MyCasesController.scala

```diff
@@ -43,8 +43,7 @@ class MyCasesController @Inject() (
       for {
         cases                         <- casesService.getCasesByAssignee(request.operator, NoPagination())
         queues: Seq[Queue]            <- queuesService.getAll
-        countQueues: Map[String, Int] <- casesService.countCasesByQueue(request.operator)
-      } yield Ok(views.html.my_cases(queues, cases.results, request.operator, countQueues))
+      } yield Ok(views.html.my_cases(queues, cases.results, request.operator, Map.empty))
```

Comment:
This controller is not used anywhere anymore but I can't delete it because removing all of the links to it is another (quite big) ticket